packages/sdks/snippets/react/src/components/CustomTabs.tsx

```diff
@@ -13,37 +13,29 @@ interface TabProps {
 export const CustomTabs = ({ tabList, builderBlock }: TabProps) => {
   const [activeTab, setActiveTab] = useState(0);
 
+  if (tabList.length === 0) return null;
+
   return (
     <>
-      {tabList?.length > 0 && (
-        <div className="tab-buttons">
-          {tabList.map((tab, index) => (
-            <button
-              key={index}
-              className={`tab-button ${activeTab === index ? 'active' : ''}`}
-              onClick={() => setActiveTab(index)}
-            >
-              {tab.tabName}
-            </button>
-          ))}
-        </div>
-      )}
-
-      <div className="tab-content">
+      <div className="tab-buttons">
         {tabList.map((tab, index) => (
-          <div
+          <button
             key={index}
-            className="tab-panel"
-            style={{ display: activeTab === index ? 'block' : 'none' }}
+            className={`tab-button ${activeTab === index ? 'active' : ''}`}
+            onClick={() => setActiveTab(index)}
           >
-            <Blocks
-              parent={builderBlock?.id}
-              path={`tabList.${index}.blocks`}
-              blocks={tab.blocks}
-            />
-          </div>
+            {tab.tabName}
+          </button>
         ))}
       </div>
+
+      <div className="tab-content">
+        <Blocks
+          parent={builderBlock?.id}
+          path={`tabList[${activeTab}].blocks`}
+          blocks={tabList[activeTab].blocks}
+        />
+      </div>
```

Comment:
the wrapping div here can go too:

```suggestion
        <Blocks
          parent={builderBlock?.id}
          path={`tabList[${activeTab}].blocks`}
          blocks={tabList[activeTab].blocks}
        />
```